src/tools/bin/plotRadiation

```diff
@@ -354,20 +405,26 @@ for myfile in args.path2Data:
     shape = shape(dataZoomed)
     if(args.logOmega):
         SP.set_xscale('log')
-        x = logspace(log10(omega_min_draw), log10(omega_max_draw), shape[1])
+        x = logspace(log10(omega_min_draw), 
+                     log10(omega_max_draw), 
+                     shape[1])
     else:
-        x = linspace(omega_min_draw, omega_max_draw, shape[1])
+        x = linspace(omega_min_draw, 
+                     omega_max_draw, 
+                     shape[1])
 
-    y = linspace(theta_min_draw, theta_max_draw, shape[0])
+    y = linspace(theta_min_draw, 
+                 theta_max_draw, 
+                 shape[0])
 
-    # plot data: (only call!!!)
+    # plot data: (this is the only command that plots data)
     cax =  SP.pcolormesh(x, y, dataZoomed,
                          shading=my_interpolation,
                          cmap=my_cmap,
                          vmax=colorbarMax,
                          norm=colorNorm)
 
-    # set x and y range
+    # set x and y range (required after pcolormesh(?))
```

Comment:
they should be always behind the `plot()`, `imshow()` and `pcolormesh()` call afaik